live-examples/html-examples/content-sectioning/css/footer.css

```diff
@@ -0,0 +1,42 @@
+.output {
+    background-color: #f9f9fa;
+}
+
+h1 {
+    font-size: 1.8em;
+    margin-bottom: 0.5em;
+}
+
+footer {
+    display: flex;
+    padding: 1em .5em;
+    margin: 1.5em 0 0;
+    font-size: 0.9em;
```

Comment:
Nit: Code style: Zero based values should omit the leading 0 ~ `0.5em` => `.5em`

Also, prefer `rem` based units for font sizes.